Buffs Toolbox.

KazeEspada opened the pull request:
Toolbox's are buffed to 15 force, putting them in line with cleavers and welder.
Syndie toolboxes are 20. Since they were literally only bought for the tools and then abandoned.

Toolboxes can also now hold double what they can now. But only with small items, such as tools. Previously they were the same size as a box.
Reasons: Due to their massive weight class, carrying them in a pain.
The other weapons at 10 force are all smaller and arguably more useful(most notably the fire extingisher).
They are uncommon.

Weapon comparisons:
Spear-10 force one handed, 18 dual wielded, fits in backslot.
Cleaver-15 force. Fits in belt, chef outfits, and backpack. Rare.
Welder-15 force. Requires fuel, only does burn, fits in pockets!

Extinguisher-10 force. Fits in backpack, useful in space, and extinguishing yourself.
Floor Tiles-10 throwforce, stacks to 50.

KazeEspada committed:
Buffs Toolbox.

Toolbox's are buffed to 15 force, putting them in line with cleavers and welder.
Reasons: Due to their massive weight class, carrying them in a pain.
The other weapons at 10 force are all smaller and arguably more useful(most notably the fire extingisher).
They are uncommon.

TechnoAlchemisto commented:
no

Fox-McCloud commented:
:-1: 

No--just no. 15 force weapons are difficult to come across for a reason.

as334 commented:
:+1: KATHUNK

octareenroon91 commented:
I support a toolbox buff.

Maybe go for 12 or 13 force if this PR fails at 15/15.

KazeEspada commented:
Dude, 15 force weapons are easy to come across...
Welders-Require fuel, FITS IN YOUR POCKET
Cleaver-Rare, Fits in your belt, AND YOUR bag.
Spears-COMMON AS FUCKING SHIT. 18 force, fits on your back.

Toolboxes are uncommon(try getting one as a medic). They don't fit in any slot. Also, they have a HUGE FUCKING INHAND sprite.

Zilenan91 commented:
12 would be more sane but 15 is okay :+1:

optimumtact commented:
I'd rather toolbox than switch

WJohn commented:
12 would be better, while you could make syndie toolboxes 15.

KazeEspada commented:
Syndie toolboxes are already 15.

MMMiracles commented:
Make it 12 so it doesn't completely negate the point of buying the syndie toolbox.

PKPenguin321 commented:
:+1: do it

KazeEspada commented:
Syndies are 20.

PKPenguin321 commented:
I'M SO READY

WJohn commented:
Yeah, but e-daggers are 18 when deployed and are twice as expensive as toolboxes.

KazeEspada commented:
Edaggers fit in your PDA. A toolbox doesn't.

Robustin commented:
People acting like spears are easy to haul around.

Welders hurt but (especially since the armor nerf) brute >>>> burn when brute is also a decent chance to knock down or knock out your victim. 

I'm on the fence. Nice seeing toolboxes be robust again but also going to get salty when some rando thwacks me unconscious in one hit because every assistant now has a really robust weapon. 

Edaggers are also contraband that will get you killed or tossed in perma. Even a bloodstained toolbox won't bother anyone.

PKPenguin321 commented:
SOUNDS LIKE YOU NEED TO H A R D E N   U P   OLDMAN

YotaXP commented:
Toolboxes used to reign supreme.

Long live the toolbox! :+1:

KazeEspada commented:
Black Toolboxes are also contraband, and now since they are one of the most dangerous weapons on the station. They might actually get you arrested.

Fox-McCloud commented:
>Welders-Require fuel, FITS IN YOUR POCKET

and they also can't knock you down.

15 force weapons have a whopping 25% chance to knock you down; there's a reason why they're rare or require effort to acquire.

KazeEspada commented:
YOU DONT EVEN GO HERE.

Zilenan91 commented:
They have more or less the same combat code as we do and fork a lot of stuff from here, so these changes do effect them.

Fox-McCloud commented:
For all intents and purposes, mechanics are pretty much the same, as are a lot of playstyles amongst players (ie: min-maxing combat efficiency); the fact base 15 has a 25% of knocking down isn't any less valid here than where I come from.

Lumipharon commented:
As above, opposed to this because high KO weapons should not be common as fuck.
Not being able to bag them doesn't matter when there are dozens all of station and they're not illegal to carry.

And making the syndie toolbox into a e-swordlite doesn't change that fact that there are ATLEAST 2 that spawn every round, so A:would be KO kings, and B: Still don't reveal anyone as traitor because there are legitimate and easy ways for non antags to get them.

PKPenguin321 commented:
Honestly, knockdown/KO could do with a bit of rework. Make it require a certain damage threshold so you can't get KOed at full health, for example.

xxalpha commented:
What's the point of increasing armor values if you're going to increase weapon damage values?

KazeEspada commented:
Storage is SUPER important. Most people wont waste their hand space with a toolbox.

KazeEspada commented:
SPEARS are MORE common. MORE POWERFUL, AND CAN BE THROWN for that sweet embed chance.

Xhuis commented:
Could we not?

AdamElTablawy commented:
No, we should.

For fuck's sake it's a bloody toolbox. It's one bloody item and it's a toolbox.

It has a giant inhand sprite, there are still tons of more robust weapons: (See-Anything that stuns like a stunprod, see-Screwdriver to eyes).

If you see some fuck running around with a giant syndie toolbox beating people it's still slow enough to kill that you're going to get robusted or called out a helluvalot faster and quicker than with an esword.

Not to mention force fifteen weapons are EXTREMELY FUCKING COMMON. Especially with the new sharpener thing that's available where you're going to get sharpened stunbatons and sharpened kitchen knives, ect. 

Because oh no, an extra +5 on weapon damage, oh no, an extra maybe sort of rng knockdown chance where chances are you're STILL going to get beat the fuck up by the guy you're attacking when he pushes you onto the floor and beats you to death with your own toolbox/stunprods you/you get mobbed for having a syndie toolbox in the first place.

If we're adding weapon sharpeners, we might as well add some decently robust toolboxes too.

If you're worried about balance-don't be, because stuns are STILL king and will beat this thing out every time, and I guarantee you that for every assistant with a toolbox that doesn't fit in their bag and they're taking up an inhand slot to use it, there's going to be four more assistants with stunprods that fit in their back and 18 damage embedding spears that are available even more commonly than toolboxes, not to mention every botany juggernaut and every scientist and engineer and security officer will have some method of defense against toolboxing, be it just applying screwdriver to eye, or flashing them, or tazing them, or just spamming deathnettle, or spamming healing chems on yourself, ect ect.

This won't really change anything-Toolboxes right now are weak-You can put all the tools in a box, in a book, in some cheese if you're feeling hungry, in a toolbelt (albiet that's the intended purpose) and just dump the box. Nobody carries toolboxes around for a reason.

Add an extra +5 to a toolbox and suddenly you have a weaker spear that has even more downsides-Doesn't fit in your bag, giant inhand sprite, 15 damage to the spear's 18-It balances itself out.

And if you're worried about syndie toolboxes-Again, you're going to get absolutely lynched for trying to kill someone with a syndie toolbox by eighty validhunting assistants. Twenty damage is certainly dangerous, but not much when your target is wearing armor of any sort/has any sort of healing chemical in them (which are very, very common) and certainly not anywhere near as dangerous nor powerful as any of the other traitor items in the syndie uplink. 

At least buffing the syndie toolbox will encourage people to actually buy the toolbox for a reason-As it stands right now we might as well just deliver all the tools in a box because the toolbox itself is just ditched in favor of literally anything else. It still has the exact same downsides + marks you as valid salad in exchange for a measly +5 brute damage where literally anything else in your traitor kit would be better.

So no, you're not causing the end of the world and destroying game balance by adding a tiny bit more force to a toolbox. You're re-implementing the thing where toolboxes used to actually be really, really robust, and were the iconic item of SS13 for a reason, you're adding more lethality for a reason, and the entire damn object balances itself out through being a giant, bulky, heavy, useless storage item where you have literally anything else in your bag that could do it's one job-holding fucking tools-better than it can, so why not at least make it decent at beating people until someone stunprods you and throws you into space?

GunHog commented:
I was against this at first, but having read the arguments for it, I am sufficiently convinced from the point of "This is probably a bad idea", to "Eh, This is okay, I guess."

KazeEspada commented:
I have also provided an alternate PR, since I realised that toolboxes are remarkably small capacity for their w_class.

KazeEspada committed:
Toolboxes now hold a bit more as well.

Toolboxes can now hold a bit more as well, but all the little compartments forces it to only hold small items(w_class 2 and smaller).

KazeEspada commented:
Buffed their carrying capacity by double, but they can still only hold small items.

Lumipharon commented:
Why do people always spew stun memes? Holding a toolbox does not stop me from also holding a stun prod.

Also people act like they've never seen someone beaten to death without a stun weapon - yes if you try toolbox a guy who has a prod, that canend very poorly, but there isnt many high cap batteries in public space (something like 2?) where as there is probable 20 toolboxes in public areas (includin maint).

KazeEspada commented:
Its about size. Toolboxes are just to bulky to be most peoples weapon of choice, one slip on a janitor's wet floor and that's 15 force to your own face.

freerealestate commented:
I like it.

YotaXP commented:
Don't forget to add the capacity change to the PR body.

This PR sets the capacity to 28 (combined w-class) from 14.  For reference, backpacks are 21 and dufflebags are 30.  This almost makes dufflebags useless.

Cheridan commented:
You're trying to compare spears and toolboxes while completely ignoring the most important thing about spears -- the fact that they're visible, lethal weapons that you have to make with intent.
Meaning that if you're walking around with one, it's clear that you're up to trouble and Sec can arrest you for carrying it around when the situation doesn't call for it.
Which is the fucking point of spears.

Toolboxes have none of these downsides. They're ubiquitous, ready-to-go, and carrying one shows no malicious intent. 

Putting a 3 damage difference between the two means you're retarded.

Durkel commented:
The time honored tradition of toolbox balance. 

Truly the most important aspect of game balance.

CPTANT commented:
:+1: 

Station needs more ways to kill people than stunning and cuffing them.

KazeEspada commented:
Cheridan confirmed for no fun.

AdamElTablawy commented:
>You're trying to compare spears and toolboxes while completely ignoring the most important thing about spears -- the fact that they're visible, lethal weapons that you have to make with intent.
Meaning that if you're walking around with one, it's clear that you're up to trouble and Sec can arrest you for carrying it around when the situation doesn't call for it.
Which is the fucking point of spears.

Toolboxes have none of these downsides. They're ubiquitous, ready-to-go, and carrying one shows no malicious intent.

Putting a 3 damage difference between the two means you're retarded.

Yes, spears are certainly more obviously lethal than reg toolboxes.

Syndie toolboxes, however, ownership of THEM demonstrates definite lethal intent, much akin to a wielded axe-Sec can arrest you for having it too.


Not to mention, if this is merged, you're definitely going to be seeing a lot more wariness around people with toolboxes-They're going to treat it the same as you running at them with a cleaver. 

Besides, what's the issue with a slight bit more lethality, anyway? Why not make it easier for someone to kill someone else when literally everything else has been safety-stapled and wrapped in bubblewrap so nobody gets their eeny weeny fingers hurt by anything but being shot by a traitor?

It's not like toolboxes being five more brute are going to make much of a difference when you can still craft superior weapons-So what if one more time out of a hundred someone gets their shit bashed in by being hit with a giant fucking heavy metal box to the face?

Cheridan commented:
>It's not like toolboxes being five more brute are going to make much of a difference when you can still craft superior weapons

except you can't. i'm not going to bother taking the time to make a "please arrest my face" spear if I already have a weapon that does nearly the same amount of damage but also holds all my tools.
Don't even mention stunprods because you can't kill with just a stunprod

Everyone pretending that more lethal weapon access is some ideal is kidding themselves. Antags have access to more deadly weapons than this so they're probably not going to be using these except out of desperation. The people who ARE gonna use these, are station loyalists who now have a perfect item to brain any syndi who comes across them. 
Which is hugbox as fuck.

Turn off the meme centers of your brains for once and think about the repercussions of shit.

KazeEspada commented:
Actually... you can. its a stun baton, so you can harm baton with it?

Xhuis commented:
@KazeEspada Stun prods do 3 or 4 damage. They're very ineffective weapons.

Iamgoofball commented:
i have no opinion on the damage argument aside from "stop comparing items that have uses other than as a weapon and claiming it's a reason to not apply buffs because it makes the item 'completely useless'"

:+1: to the storage increase, toolboxes are useless **in the storage department** compared to belts and literal cardboard boxes right now

KorPhaeron commented:
Sounds like we should nerf cardboard boxes instead

Making it so people can't fit both the entire armory+all of medbay in their bag at once might force some co-operation but I'm not sure the backlash is worth dealing with.

Lumipharon commented:
Considering that toolboxes ARE bulky items, it really does just make sense to give them some more storage space.

Zilenan91 commented:
Just make energy guns unable to fit into boxes and it's good

KorPhaeron commented:
Having dangerous non-antag weapons around the station is nice because it gives you a way to break things without carrying a round big meta VALID flags on you

Xhuis commented:
Energy guns have never been able to fit into boxes. They're both normal-sized, and IIRC a storage item can't hold anything that is equal to or bigger than its own size.

KazeEspada commented:
Storage hold whatever it damn pleases. Its own size has no effect.

Xhuis commented:
Backpacks can't hold bulky items. Boxes can't hold normal items, etc. It doesn't hold whatever it pleases, it can't hold stuff that's as large as itself.

Lumipharon commented:
sec belts can hold normal sized batons.

Check and/or mate.

Xhuis commented:
Yeah, smuggler satchels can hold normal stuff, and so can sec belts. You got me there. My main point is that the most common storage - i.e., stuff that's available pretty much everywhere and has seven slots, such as the toolbox - is typically limited as stated above.

octareenroon91 commented:
Sec belts and toolbelts have a whitelist of items to store: `can_hold`

Ultimate-Chimera commented on code/game/objects/items/weapons/storage/toolbox.dm:
```diff
@@ -73,8 +74,8 @@
-	force = 15
-	throwforce = 18
+	force = 20
+	throwforce = 20
```
Shouldn't the throwforce be +5 of what it used to be? So maybe 23.

KazeEspada replied:
shhh, i secretly hate throwforce.

KazeEspada commented on code/game/objects/items/weapons/storage/toolbox.dm:
```diff
@@ -5,11 +5,12 @@
 	throw_speed = 2
 	throw_range = 7
 	w_class = 4
+	max_combined_w_class = 28
```
this determines how much it can hold, theres another var that determines what size.
BTW, toolboxes can only hold size 2 or smaller. Which is pockets.

KorPhaeron commented:
Cheridan said no like 2 weeks ago
